examples/demo-prover/methods/guest-mock/riscv32im-risc0-zkvm-elf.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I think we can delete this file too. That's also a holdover from the old Risc0 version

Reply:
fixed